### ISSUE TITLE
GPX Importer not taking platform name

### ISSUE THREAD

**Issue by IanMayo** [debrief/pepys-import#390]
The sample GPX file does include the platform name:
![image](https://user-images.githubusercontent.com/1108513/82306420-f0b96180-99b6-11ea-9bb4-8f7ff720f380.png)

When it's provided, we do not need to ask the user for it.

**Comment by robintw** [comment]
The GPX used to read the name from the file, but this was removed based on the spreadsheet you wrote in #350, which said that the name should be resolved by the resolver.

I can reinstate the code I took out, which means it will read the name from the file. In the current implementation, it will then ask whether you want to search for a platform named NELSON or add a new platform.

We can add the further details about the platform identity (type, pennant number etc) to the interface fairly easily - but I think we should wait until we have the information from the client that we've asked for in #379 about what uniquely defines a platform, as otherwise we'll just have to redo this when we decide what uniquely identifies one.
